apps/next/Dockerfile.worker

```diff
@@ -0,0 +1,20 @@
+FROM debian:bookworm-slim
+SHELL ["/bin/bash", "-c"]
+RUN apt update && apt upgrade -y
+RUN apt install curl unzip -y
+
+# install node 20
+RUN curl -sL https://deb.nodesource.com/setup_20.x | bash -
+RUN apt update && apt upgrade -y
+RUN apt install nodejs -y
+
+# install bun
+RUN curl -fsSL https://bun.sh/install | bash
+ENV PATH "$PATH:/root/.bun/bin"
+
```

Comment:
@ellipsis-dev There are a bunch of other things I need installed in this image, namely:

- kubectl
- clusterctl
- wget
- kustomize
- helm
- argo rollouts kubectl plugin https://argo-rollouts.readthedocs.io/en/stable/installation/
- jq
- yq
- mc command line https://min.io/docs/minio/linux/reference/minio-mc.html
- htpasswd

can you do this?